src/main/java/com/cloudbees/hudson/plugins/folder/AbstractFolder.java

```diff
@@ -305,6 +306,15 @@ protected AbstractFolderViewHolder newFolderViewHolder() {
         return new DefaultFolderViewHolder(views, null, newDefaultViewsTabBar());
     }
 
+    @Override
+    public String getIconClassName() {
+        // avoid https://issues.jenkins.io/browse/JENKINS-74990
+        if (icon.getClass().getName().equals("jenkins.branch.MetadataActionFolderIcon")) {
```

Comment:
not ideal but I couldn't see a cleaner solution.

This if block means that the `MetadataActionFolderIcon` falls back into the code it previously did before `Folder` had an `IconSpec`.

in command palette it works fine because e.g. `OrganizationFolder` overrides `getIconClassName` so it gets the right icon, in the example I tried (GitHub organisation) it didn't load the icon in command palette but that's because it was using the `IconSet` API and not symbols which isn't supported by command palette right now (cc @janfaracik )